### PR TITLE
Include Deno.WriteFileOptions

### PR DESCRIPTION
**Pull request by jsejcksn**

### CODE REVIEW

**Comment by jsejcksn** [comment]
This should also fix the formatting check in th PR.

File: std/fs/write_json.ts [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
-function serializeToJsonFileContent(
+function serialize(
   filePath: string,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   object: any,
   options: WriteJsonOptions,
 ): string {
   try {
-    const jsonContent = JSON.stringify(
+    let jsonString = JSON.stringify(
```
Comment (tranzystorek-io) [inline std/fs/write_json.ts:17]: I'm not sure why `jsonString` here is a `let` instead of `const`

Comment (jsejcksn) [reply std/fs/write_json.ts]: Good catch! Fixed. (It was leftover from a previous mutable operation that I refactored.)

**Comment by tranzystorek-io** [comment]
Thank you for your contribution @jsejcksn

**Pull request merged**